portfolio/src/main/java/com/google/sps/servlets/CountryServlet.java

```diff
@@ -34,20 +34,82 @@ public void init() {
     }
 
     public void countryInit() {
-        countries.add(new Country("Germany", "Berlin"));
-        countries.add(new Country("Brazil", "Brasilia"));
+        // South America
+        countries.add(new Country("Brazil", "Brasília"));
+        countries.add(new Country("Argentina", "Buenos Aires"));
+        countries.add(new Country("Colombia", "Bogotá"));
+        countries.add(new Country("Venezuela", "Caracas"));
+        countries.add(new Country("Uruguay", "Montevidéu"));
+
+        // Central America
+        countries.add(new Country("Panama", "Panama City"));
+        countries.add(new Country("Guatemala", "Guatemala City"));
+        countries.add(new Country("Haiti", "Port-au-Prince"));
+        countries.add(new Country("Cuba", "Havana"));
+        countries.add(new Country("Dominican Republic", "Santo Domingo"));
+        
+        // North America
         countries.add(new Country("United States", "Washington"));
-        countries.add(new Country("France", "Paris"));
         countries.add(new Country("Canada", "Ottawa"));
+        countries.add(new Country("Mexico", "Mexico City"));
+        
+        // Asia
         countries.add(new Country("Russia", "Moscow"));
+        countries.add(new Country("India", "New Delhi"));
+        countries.add(new Country("Pakistan", "Islamabad"));
+        countries.add(new Country("Japan", "Tokyo"));
+        countries.add(new Country("China", "Beijing"));
+        
+        // Africa
         countries.add(new Country("South Africa (any one of the 3 capitals)", "Bloemfontein, Cape Town, Pretoria"));
+        countries.add(new Country("Morocco", "Rabat"));
+        countries.add(new Country("Egypt", "Cairo"));
+        countries.add(new Country("Algeria", "Algiers"));
+        countries.add(new Country("Kenya", "Nairobi"));
+
+        // Europe
+        countries.add(new Country("France", "Paris"));
+        countries.add(new Country("Germany", "Berlin"));
+        countries.add(new Country("England", "London"));
+        countries.add(new Country("Wales", "Cardiff"));
+        countries.add(new Country("Ireland", "Dublin"));
+        countries.add(new Country("Northern Ireland", "Belfast"));
     }
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        Random rand = new Random();
+        List<Country> sublist = sublistOfCountries(10, countries);
         response.setContentType("text/html;");
-        response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+        response.getWriter().println(convertToJsonUsingGson(sublist));
+        // response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+    }
+
+    public List<Country> sublistOfCountries(int size, List<Country> countriesOriginal) {
+        List<Country> countries = new ArrayList<>(countriesOriginal);
+        int listSize = countries.size();
+
+        if (size > listSize) {
+            return null;
```

Comment:
returning `null` here is "not very nice" as it's likely going to cause an exception when the calling code doesn't perform null checks.

Less fragile:
 1) `throw new IllegalStateException("cannot return sublist larger than original");` -- closest to returning null
 2) `return countries;` -- best effort, return as many as we can
 3) `return Collections.emptyList();` surprising, but won't cause NullPointerExceptions.